templates/website-footer.md

```diff
@@ -1,32 +1,32 @@
 # Copyright notices for project website footers
 
-For those building or managing a website for a Commonhaus Foundation project, please incorporate one of the following standard footers.
+For those building or managing a website for a Commonhaus Foundation project, you should include mention of the Commonhaus Foundation in your website footers. Some examples are provided below (replace the links to YOUR PROJECT to point to the full URL of the main landing page for your project's website).
```

Comment:
Are we requiring a Commonhaus footer, or only prefer it?

Reply:
I would like to require mention of some kind, but not necessarily this exact text or logo usage. The Trademark text should be there.

Reply:
If we want to require usage, should the wording be:

```suggestion
For those building or managing a website for a Commonhaus Foundation project, you must include mention of the Commonhaus Foundation in your website footers. Some examples are provided below (replace the links to YOUR PROJECT to point to the full URL of the main landing page for your project's website).
```